R/as_curve_params.R

```diff
@@ -0,0 +1,43 @@
+#' Load antibody decay curve parameter
+#'
+#' @param data a [data.frame()] or [tibble::tbl_df]
+#' @param antigen_isos [character()] vector of antigen isotypes to be used in analyses
+#' @returns a `curve_data` object (a [tibble::tbl_df] with extra attribute `antigen_isos`)
+#' @export
+#' @examples
+#' curve_data <- load_curve_params("https://osf.io/download/rtw5k/")
```

Comment:
this example still uses `load_curve_params()` instead of `as_curve_params()`; please update.